sys/shell/src/shell_os.c

```diff
@@ -249,6 +268,13 @@ static const struct shell_cmd os_commands[] = {
         .sc_cmd_func = shell_os_reset_cmd,
 #if MYNEWT_VAL(SHELL_CMD_HELP)
         .help = &reset_help,
+#endif
+    },
+    {
+        .sc_cmd = "ls_dev",
```

Comment:
Just a thought - what do you think about renaming this to `lsdev` (without the underscore)?  That more closely mimics `lshw`,`lspci`, etc. from Unix.

Reply:
That should be ok.